tiledb/ml/models/sklearn.py

```diff
@@ -72,10 +99,30 @@ def _create_array(self):
             schema = tiledb.ArraySchema(
                 domain=dom,
                 sparse=False,
-                attrs=attrs,
+                attrs=attrs, ctx=tiledb_create_context
             )
 
             tiledb.Array.create(self.uri, schema)
+
+            if self.tiledb_uri:
+                time.sleep(0.25)
```

Comment:
`sleep()` is a code smell in general; at the very least it merits a comment of why it is necessary (if it is necessary)

Reply:
It seems not easy to get rid of it ran several tests there is the possibility to fail even with this delay for this scenario, used in cloud notebooks with success until now

Reply:
Comment added